scripts/release/update-package-versions.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+// Updates `@ckeditor/ckeditor5-*` and `ckeditor5` dependencies in `packages/*` and `release/*` directories to the latest version.
+// Changes in `packages/*` will be committed as well.
+//
+// See https://github.com/cksource/ckeditor5-internal/issues/1123.
+//
+// This task must be called before: `npm run release:publish`.
+//
+// Use:
+// npm run release:update-package-versions -- --dry-run --diff
+//
+// Available arguments:
+// --dry-run:   Prevents the script from committing, and instead shows list of files that were changed in `packages/*` directory.
+// --diff:      Instead of list of files, it displays detailed list of changes from each file. Has no effect without --dry-run.
```

Comment:
Please, merge these options. While the `--dry-run` option makes sense and allows testing what the script will do, I don't understand how it should work when executing `npm run release:update-package-versions -- --diff`.